themes/bootstrap3/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "vufind-bootstrap3",
+  "version": "9.1.0",
```

Comment:
Is the version property absolutely required to make things work, or is it only required in order to publish a package? Since this is just a utilitarian sub-part of the overall project, I would be inclined to reduce the amount of metadata here. I'd just leave name and description, but get rid of version, repository and bugs -- keeping that data here just invites it to get outdated / out of sync with everything else. If we absolutely have to keep version, I'd be inclined to use 0.0.0 as you suggest, so we don't have to remember to update it with every release.

Reply:
At least update works without version and the other fields, so let's go with that and add them back later if necessary.